src/lib.rs

```diff
@@ -111,7 +111,6 @@
 // [1]: Mind that the waker only takes zero bytes when all features are disabled, making it
 //      impossible to *wait* for the message. `try_recv` the only available method in this scenario.
 
-#![allow(unexpected_cfgs)]
```

Comment:
Does the lint definition in `Cargo.toml` not help with the tests? In total the codebase has six `#![allow(unexpected_cfgs)]` that it would be awesome if we could get rid of also.

Reply:
Oh I'm sorry I didn't even see those! Yeah, of course the lint table makes those unnecessary as well (:
Fixed